.github/workflows/package-pusher.yml

```diff
@@ -0,0 +1,59 @@
+name: Package Pusher
+
+on:
+  workflow_dispatch:
+    inputs:
+      packages:
+        description: "What is the packages you wish to push (seperate package names with a whitespace)?"
+        required: true
+
+jobs:
+  pusher:
+    runs-on: windows-2019
+    env:
+      github_user_repo: ${{ github.repository }}
+      API_KEY: ${{ secrets.CHOCO_API_KEY }}
+      SOURCE_LOCATION: "https://push.chocolatey.org"
+
+    steps:
+      - name: Install Dependencies
+        run: |
+          # Placeholder for installing dependencies
+          # choco install chocolatey-core.extension -y
+        shell: powershell
+      - name:
```

Comment:
Is there a reason that this doesn't have a name?

Reply:
I have added a name for this one now.